.circleci/config.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - run: ./install_ffmpeg.sh
       - run: go get github.com/golang/glog
       - run: go get github.com/ericxtang/m3u8
+      - run: go get github.com/aws/aws-sdk-go/aws
```

Comment:
I wonder if we can `go get ./...` all of these (here and in the Dockerfile). Do we need to be explicit?